packages/controllers/profile_edit.coffee

```diff
@@ -0,0 +1,29 @@
+if Meteor.isClient
+  Template.profileEdit.onCreated ->
+    instance = this
+    subscription = instance.subscribe('userProfile')
+
+  Template.profileEdit.helpers
+    userProfile: ->
+      UserProfiles.findOne()
+
+  Template.profileEdit.events
+    'submit form': (event) ->
+      event.preventDefault()
+      form = event.target
+      fields = {
+        fullName: form.fullName.value
+        jobTitle: form.jobTitle.value
+        bio: form.bio.value
+        emailHidden: form.emailHidden.value
+      }
+      Meteor.call('updateProfile', form.profileId.value, fields, -> {})
+
+if Meteor.isServer
```

Comment:
I like the idea that the server method is defined in the same place where it's used and that the publication and subscription are in the same file, but it's a little weird to load a view controller on both the client and the server so I could be convinced that this should go somewhere else.